.github/workflows/integration.yml

```diff
@@ -153,7 +153,56 @@ jobs:
       run: ./gradlew lintFrontend
 
     - name: Build with Gradle
-      run: time ./gradlew clean checkstyleAll test systemTest coverage
+      run: ./gradlew clean checkstyleAll test systemTest coverage
+
+  windows:
+    name: ${{ matrix.os }} JDK 8
+    strategy:
+      matrix:
+        os: [windows-2022, windows-2019]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    - name: Verify Java version
+      run: Write-Output "Actual JDK in use -- "; java -version
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
```

Comment:
This needs to be updated too now that #1714 has been merged.

Reply:
Updated!